src/ontology/mondo-edit.obo

```diff
@@ -158291,7 +158293,7 @@ synonym: "palmoplantar keratoderma, epidermolytic, with knuckle pads" RELATED [O
 synonym: "palmoplantar keratoderma, Vorner type" RELATED [OMIM:144200]
 synonym: "Ppke" RELATED [OMIM:144200]
 synonym: "tylosis" RELATED [OMIM:144200]
-xref: DOID:0080223 {source="MONDO:equivalentTo"}
```

Comment:
I see this is an xref on MONDO:0968949 'palmoplantar keratoderma, epidermolytic', should it be equivalent on this term?

Reply:
This requires a little more investigation. So I left it for next month :-)  (not because I am lazy, but because I don't have the focus time needed to make sure these are actually the same concept)

Reply:
I take that back! I did add it but didn't have the MONDO:equivalentTo. 
It is now resolved.